test/Sentry.DiagnosticSource.Tests/SentryEFCoreListenerTests.cs

```diff
@@ -17,17 +17,13 @@ public class SentryEFCoreListenerTests
         => type switch
         {
             _ when
-                type == EFQueryCompiling ||
-                type == EFQueryCompiled
+                type is EFQueryCompiling or EFQueryCompiled
```

Comment:
The `_ when type is` syntax is extraneous here. The patterns can just be used directly.  Alternatively, the code might be more readable as a traditional switch/case statement with fall-through.